app-modules/student-data-model/src/Filament/Resources/StudentResource.php

```diff
@@ -78,6 +78,7 @@ public static function getGlobalSearchResultDetails(Model $record): array
             'Email Address' => collect([$record->email, $record->email_id])->filter()->implode(', '),
             'Mobile' => $record->mobile,
             'Phone' => $record->phone,
+            'Preferred Name' => $record->preferred,
```

Comment:
You also need to add `preferred` to `getGloballySearchableAttributes`.

Also, @danharrin should we include it in the `scoreGlobalSearchResults` in `modifyGlobalSearchQuery` in any way as well?

Reply:
Fixed

Reply:
As above